chapters/planning-data-work.tex

```diff
@@ -483,26 +456,33 @@ \subsection{Documenting and organizing code}
 They all come from the principle that code is an output by itself,
 not just a means to an end,
 and should be written thinking of how easy it will be for someone to read it later.
+At the end of this section, we include a template for a Master Script in Stata, 
+to provide a concrete example of the required elements and structure.
+Throughout this section, we refer to lines of the example .do file 
+to give concrete examples of the required code elements, organization and structure.  
 
 Code documentation is one of the main factors that contribute to readability.
 Start by adding a code header to every file.
 A code header is a long \textbf{comment}\sidenote{
   \textbf{Comments:} Code components that have no function,
   but describe in plain language what the code is supposed to do.
 }
-that details the functionality of the entire script.
+that details the functionality of the entire script;
+refer to lines 5-10 in the example .do file. 
 This should include simple things such as
 the purpose of the script and the name of the person who wrote it.
 If you are using a version control software,
 the last time a modification was made and the person who made it will be recorded by that software.
 Otherwise, you should include it in the header.
-Finally, use the header to track the inputs and outputs of the script.
+You should always track the inputs and outputs of the script, as well as the uniquely identifying variable;
+refer to lines 49-51 in the example .do file. 
 When you are trying to track down which code creates which data set, this will be very helpful.
 While there are other ways to document decisions related to creating code,
 the information that is relevant to understand the code should always be written in the code file.
 
 In the script, alongside the code, are two types of comments that should be included.
-The first type of comment describes what is being done.
+The first type of comment describes what is being done;
+refer to line 35 in the example .do file.
```

Comment:
```suggestion
refer to line 35 in the example do file.
```

Reply:
Agree on how to write do-file

Reply:
Official usage is `do-files` so we should use that throughout.